### PR TITLE
Unittest/added registration tests

### PR DESCRIPTION
**Pull request by edward0802**

### CODE REVIEW

**Review by eugent482 — APPROVED** [review]

**Review by OleksandrAntoniuk — APPROVED** [review]

File: InternetHospital.UnitTests/TestHelpers/RegistrationHelper.cs [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+            return fileMock;
+        }
+
+        public static User GetNewPatient(Fixture fixture)
+        {
+            return fixture.Build<User>()
+                .With(p => p.StatusId, NEW_USER_STATUS)
+                .With(p => p.Status, new Status
+                {
+                    Id = 2,
```
Comment (TREN-D) [inline InternetHospital.UnitTests/TestHelpers/RegistrationHelper.cs]: You declared const value STATUS_ID why are you not using it here?

File: InternetHospital.UnitTests/TestHelpers/RegistrationHelper.cs [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        }
+
+        public static User GetNewDoctor(Fixture fixture)
+        {
+            return fixture.Build<User>()
+                .With(u => u.Id, 2)
+                .With(u => u.StatusId, NEW_USER_STATUS)
+                .With(u => u.Status, new Status
+                {
+                    Id = 2,
```
Comment (TREN-D) [inline InternetHospital.UnitTests/TestHelpers/RegistrationHelper.cs]: And here.

File: InternetHospital.UnitTests/SignUp/SignUpServiceUnitTests.cs [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+using InternetHospital.UnitTests.TestHelpers;
+using Microsoft.AspNetCore.Http;
+using Moq;
+using Xunit;
+
+namespace InternetHospital.UnitTests.SignUp
+{
+    public class SignUpServiceUnitTests
+    {
+        const int NEW_USER_STATUS = 2;
```
Comment (TREN-D) [inline InternetHospital.UnitTests/SignUp/SignUpServiceUnitTests.cs]: Unused field.

File: InternetHospital.UnitTests/SignUp/RegistrationServiceUnitTests.cs [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+                // asset
+                res.Email.Should().BeEquivalentTo(expectedUser.Email);
+            }
+        }
+
+        [Fact]
+        public async void ShoulDoctorRegistrate()
+        {
+            const int USER_ID = 2;
```
Comment (TREN-D) [inline InternetHospital.UnitTests/SignUp/RegistrationServiceUnitTests.cs]: Is it okay to declare USER_ID here? You already have helper class with this field, all you need is to make him public.

File: InternetHospital.UnitTests/SignUp/RegistrationServiceUnitTests.cs [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+            const string DOCTOR = "Doctor";
```
Comment (TREN-D) [inline InternetHospital.UnitTests/SignUp/RegistrationServiceUnitTests.cs]: In addition you have const PATIEN and DOCTOR in all methods, maybe its better to remove them into helper class?

**Review by TREN-D — APPROVED** [review]

**Pull request merged**